cpp/dolfinx/mesh/graphbuild.h

```diff
@@ -23,26 +23,29 @@ namespace dolfinx::mesh
 enum class CellType;
 
 /// @brief Compute the local part of the dual graph (cell-cell
-/// connections via facets) and facet with only one attached cell.
+/// connections via facets) and facets with only one attached cell.
 ///
-/// @param[in] celltype Cell type.
-/// @param[in] cells Cell vertices (stored as as a flattened list).
+/// @param[in] celltypes List of cell types.
+/// @param[in] cells Lists of cell vertices (stored as flattened lists for each
+/// cell type).
 /// @return
 /// 1. Local dual graph
 /// 2. Facets, defined by their vertices, that are shared by only one
-/// cell on this rank. The logically 2D is array flattened (row-major).
+/// cell on this rank. The logically 2D array is flattened (row-major).
 /// 3. The number of columns for the facet data array (2).
 /// 4. The attached cell (local index) to each returned facet in (2).
 ///
 /// Each row of the returned data (2) contains `[v0, ... v_(n-1), x, ..,
 /// x]`, where `v_i` is a vertex global index, `x` is a padding value
 /// (all padding values will be equal).
 ///
-/// @note The return data will likely change once we support mixed
-/// topology meshes.
+/// @note The cell indexing stacks the cells from each cell type consecutively,
```

Comment:
I don't follow the comment - how does it map to the return arguments? A short example would help.